nginx.conf

```
@@ -40,6 +40,15 @@ http {
     # HSTS (ngx_http_headers_module is required) (31536000 seconds = 12 months)
     add_header Strict-Transport-Security max-age=31536000;
 
+    # CSP (Content-Security-Policy)
+    set $csp "default-src 'self' api.epa.gov; script-src 'self' 'nonce-$request_id' https://*.googletagmanager.com https://dap.digitalgov.gov https://script.crazyegg.com; style-src 'self' 'unsafe-inline' https:; img-src 'self' api.epa.gov data: www.googletagmanager.com https://*.google-analytics.com https://*.googletagmanager.com; font-src 'self' https:; connect-src 'self' api.epa.gov www.googletagmanager.com https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://script.crazyegg.com;";
+    # Content-Security-Policy (for Chrome 25+, Firefox 23+, Safari 7+)
+    add_header Content-Security-Policy $csp;
+    # X-Content-Security-Policy (for Firefox 4.0+ and Internet Explorer 10+)
+    add_header X-Content-Security-Policy $csp;
+    # X-WebKit-CSP (for Chrome 14+ and Safari 6+)
+    add_header X-WebKit-CSP $csp;
+
```

Add the replacement logic here:
```suggestion

    sub_filter_once off;
    sub_filter "###NONCE###" $request_id;
```